README.md

```diff
@@ -208,46 +208,46 @@ Team management requires a valid read-write API key in your configuration file.
 ####Get Teams
 Get all teams associated with this accounts.
 
-     lecli getteams
+     lecli get teams
      
 ####Get a Specific Team
 Get a specific team by providing team UUID.
 
-    lecli getteam <team id>
+    lecli get team <team id>
     
 ####Create a New Team
 Create a new team with the given name.
 
-    lecli createateam <new name>
+    lecli create team <new name>
     
 ####Delete a Team
 Delete a team with the given UUID.
 
-    lecli deleteteam <team id>
+    lecli delete team <team id>
     
 ####Rename a Team
 Rename a team with the given UUID to given name.
 
-    lecli renameteam <team id> <team name>
+    lecli rename team <team id> <team name>
 
 ####Add User to a Team
 Add a new user to a team with the given UUID and user UUID respectively.
 
-    lecli addusertoteam <team id> <user key>
+    lecli update team_with_user <team id> <user key>
```

Comment:
"update team_with_user" is I think a bit ambiguous, maybe `lecli update team add_user` is much cleaner, what do you think?